frontend/src/components/activePage/ActiveLanding/ViewModeButton/ViewModeButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC, Dispatch, SetStateAction } from 'react';
+import { Radio } from 'antd';
+
+export interface IViewModeButtonProps {
+  textUsr: string;
+  textMngr: string;
```

Comment:
Do we have any actual need to have these passed from outside? 
(I guess I'm missing something as my usual 🥴)

Reply:
Not really, I just thought it would be neat to be able to change the text through a prop.